packages/core/graphql/package.json

```diff
@@ -7,8 +7,7 @@
   "module": "dist/index.module.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "dev": "yarn gen && parcel watch",
```

Comment:
Does anyone use the watch command? I don't. If so, we can reintroduce the tsc equivalent?